### DISCUSSION TITLE
Display metadata from different device

### DISCUSSION THREAD

**Discussion by Hk1020** [mikebrady/shairport-sync#1912] [Q&A] [answered]
I’ve been wondering about something similar to the metadata display but which will show me what is playing on a different device. Eventually I’d like to have some display (eg raspi) that shows what is playing on the homepod. I found several projects which read and display what the current shairport is playing. So if I play on the homepod and the raspi (shairport) simultaneously I can get what I want.  But is there a way to get there without having to play on an additional device?

**Comment by mikebrady** [comment, marked as answer]
Thanks for the question. Some of the AirPlay remote control apps seem to be able to get this kind of information remotely -- one source of information might be [pyatv](https://pyatv.dev).

**Reply by Hk1020** [reply]
Thank you very much. This is a very good starting point. I could get the information with the provided command line tool. With some polishing I could certainly make what I want if not someone else has already done it.
